File: src/Clients/Web/SamStoreWeb/src/app/models/ShoppingCarts/shopping-cart-item.DTO.ts
export class ShoppingCartItemDTO {
  public id?: string;
  public cartId?: string;
  public productId: string;
  public name: string;
  public quantity: number = 1;
  public price: number = 0;
  public imagePath: string = "";

  /**
   *
   */
  constructor(productId: string, name: string) {
    this.productId = productId;
    this.name = name;
  }
}

File: src/ApiGateways/SamStore.BFF.Orders/Models/ShoppingCartItemDTO.cs
namespace SamStore.BFF.Orders.Models
{
    public class ShoppingCartItemDTO
    {
        public Guid Id { get; set; }
        public Guid CartId { get; set; }
        public Guid ProductId { get; set; }
        public string Name { get; set; }
        public int Quantity { get; set; }
        public decimal Price { get; set; }
        public string? ImagePath { get; set; }
    }
}
